src/_includes/components/apply-note.njk

```diff
@@ -0,0 +1,21 @@
+<div style="display: flex;">
```

Comment:
@will0101 For this PR we only want to focus on content, no style. Let's remove this `style` attribute

Reply:
Removed